evm/src/keccak_sponge/keccak_sponge_stark.rs

```diff
@@ -382,12 +379,8 @@ impl<F: RichField + Extendable<D>, const D: usize> Stark<F, D> for KeccakSpongeS
         // Ensure that full-input block and final block flags are not set to 1 at the same time.
         yield_constr.constraint(is_final_block * is_full_input_block);
```

Comment:
Looks like this is now the same as the next constraint below, we can remove one or the other :)

Reply:
Updated!